README.md

```diff
@@ -1,46 +1,67 @@
 # Generate SBOM GitHub Action
 
-This action generates a Software Bill of Materials (SBOM) from git repositories using [Syft](https://github.com/anchore/syft).
+[![GitHub release](https://img.shields.io/github/release/scality/sbom.svg)](https://github.com/scality/sbom/releases/latest)
+[![License: Apache-2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/anchore/sbom-action/blob/main/LICENSE)
 
-Syft will search in the entire repository and generate a sbom file for each language.
+A GitHub Action for creating a software bill of materials (SBOM)
+using [Syft](https://github.com/anchore/syft).
 
-Actually, only those languages are supported:
+## Basic Usage
 
-- python
-- go
-- javascript
-- github-actions
+```yaml
+- uses: scality/sbom@v1
+  target: ./
+```
 
-If you need to add more syft cataloger, please open an issue.
+This will create SBOM result files based on type, ex: 
 
-## Inputs
+- repo_sbom_v1.1.0-4-gd6cdf1f.json
+- image_nginx_latest.json
+- iso_metalk8s.iso_128.json
 
-### `ref`
+## Example Usage
```

Comment:
In the tests, I had to specify `fetch-depth` and `fetch-tags` parameters when checking out the repo because of a `git describe` command that is ran, otherwise it would fail. 

Might be good to specify that somewhere in this doc:
```yaml
# Repository must be cloned with the following parameters
- uses: actions/checkout@v4
  with:
    fetch-depth: 0
    fetch-tags: true
```